plotnine/tests/test_geom_text_label.py

```diff
@@ -49,3 +49,37 @@ def test_label_aesthetics():
          scale_y_continuous(limits=(-0.5, n-0.5)))
 
     assert p == 'label_aesthetics'
+
+
+def test_adjust_text():
+    from plotnine.data import mtcars
```

Comment:
Put it at top of file.

Reply:
I presume you mean the import and not the test.